### ISSUE TITLE
Installation from .ova has no host configured for katana.wtf

### ISSUE THREAD

**Issue by Verdoso** [SamuraiWTF/samuraiwtf#187]
Hi there,
I just did an installation from the current .ova fiule, v5.3, and after installation, I browse to http://katana.wtf and all I get is the Ngnix default page "Welcome to ngnix!". On the other hand, trying with https://katana.wtf returns an "Unable to connect" error. 

I have checked /etc/ngnix/sites-available and /etc/ngnix/sites-enabled and the only enabled site is default with a basic default configuration with no katana configuration whatsoever. I can see there is a /etc/ngnix/conf.d/katana.test.conf file to configure a proxy for that name to localhost:8087, but I can see no similar configuration for the hostname katana.wtf.

The python service is indeed listening at port 8087 and I can access it directly from the browser.

I tried katana --update and nothing changed, still no katana at port 80/443.

Is there something I'm missing? As far as the instructions go, you should be able to access katana at the https://katana.wtf address, right? It looks as if some configuration files are missing.

Thank you!

**Comment by mgillam** [comment]
Thanks for reporting this! Can you access it via `http://katana.test` by any chance? The project was moving toward favoring .test instead of the .wtf tld. I'm wondering if the nginx config is only for .test but /etc/hosts still has the .wtf entry.

**Comment by Verdoso** [comment]
Yes, I can. It jumps automatically to https://katana.test:8443/ and gives an error about the certificate, that you can skip going to advanced and then going on.

If that's the new way moving forward, then maybe the documentation should be updated to reflect the new name and warn users about the certificate message? That would prevent users opening new issues like this one 😄 

Thank you for the prompt response!
Cheers,
D
